### PR TITLE
Added new districts for the country nepal

### PR DESCRIPTION
**Pull request by shankhadevpadam**
### What is the reason for this PR?
Added few new districts for the country name Nepal.

### Author's checklist

- [x] Follow the [Contribution Guide](https://github.com/FakerPHP/Faker/blob/main/.github/CONTRIBUTING.md)

### Summary of changes
Just added new districts.

### Review checklist

- [x] All checks have passed
- [ ] Changes are approved by maintainer

### CODE REVIEW

**Review by bramceulemans — APPROVED** [review]

**Comment by bramceulemans** [comment]
I checked the mentioned source, and they all seem good to me. Thanks!

**Review by bramceulemans — CHANGES_REQUESTED** [review]
Make sure the checks are passing, though.

File: src/Faker/Provider/ne_NP/Address.php [hunk 1]
```diff
@@ -19,20 +19,22 @@ class Address extends \Faker\Provider\Address
...
+        'Nawalpur', 'Nuwakot',
         'Okhaldhunga',
-        'Palpa', 'Panchthar', 'Parbat', 'Parsa', 'Pyuthan',
-        'Ramechhap', 'Rasuwa', 'Rautahat', 'Rolpa', 'Rukum', 'Rupandehi',
+        'Palpa', 'Panchthar', 'Parasi', 'Parbat', 'Parsa', 'Pyuthan',
+        'Ramechhap', 'Rasuwa', 'Rautahat', 'Rolpa', 'Rupandehi',
         'Salyan', 'Sankhuwasabha', 'Saptari', 'Sarlahi', 'Sindhuli', 'Sindhupalchok', 'Siraha', 'Solukhumbu', 'Sunsari', 'Surkhet', 'Syangja',
         'Tanahu', 'Taplejung', 'Terhathum',
         'Udayapur',
+        'Western Rukum'
```
Comment (bramceulemans) [inline src/Faker/Provider/ne_NP/Address.php:37]:
```suggestion
        'Western Rukum',
```

This should fix the checks.

**Review by pimjansen — APPROVED** [review]

**Comment by pimjansen** [comment]
Lgtm

**Review by bramceulemans — APPROVED** [review]

**Pull request merged**